### PR TITLE
simorgh dependabot updates

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves

#2724 
#2728 
#2747 
#2803 
#2804 
#2833 
#2858 
#2871 
#2872 
#2636 
#2650 
#2538 
#2941 
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — APPROVED** [review]
LGTM, touches prod deps so will need testing

**Review by dr3 — CHANGES_REQUESTED** [review]
Snapshots failing and conflicts

**Comment by oluoluoxenfree** [comment]
@RayNjeri has done a few of these so I'll remove the overlapping ones ( #2863 )

File: package.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
   },
   "homepage": "https://github.com/bbc/simorgh#readme",
   "dependencies": {
-    "@bbc/gel-foundations": "3.1.0",
+    "@bbc/gel-foundations": "3.2.0",
```
Comment (jamesbhobbs) [inline package.json]: despite the name I believe this is a psammead package, can we leave it out of this PR please

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
Also the PR description doesn't list as many issues closed as are in the diff, can you update it please? (and close those PRs as you add them to the PR description)

**Review by jamesbhobbs — APPROVED** [review]

**Comment by oluoluoxenfree** [comment]
Didn't update audit-ci due to #2955

**Review by dr3 — APPROVED** [review]
LGTM, general build and page view regression IMO. Make sure all builds & loads and is visually the same

**Comment by jamesbhobbs** [comment]
Manual testing done by Priya and i

**Pull request merged**